input/intro-notes/StructureDefinition-au-core-procedure-notes.md

```diff
@@ -41,19 +41,19 @@
         <td>code</td>
         <td><b>MAY</b></td>
         <td><code>token</code></td>
-        <td>The requester <b>SHALL</b> provide at least a code value and <b>MAY</b> provide both the system and code values. The responder <b>SHALL</b> support both.</td>
+        <td>The requester <b>SHALL</b> provide at least a code value and <b>MAY</b> provide both the system and code values. The responder <b>SHALL</b> support both.<br/><br/>The requester <strong>SHALL</strong> support <code>multipleOr</code>. The responder <strong>SHALL</strong> support <code>multipleOr</code>.</td>
```

Comment:
Should be '<strong>SHOULD</strong> support <code>multipleOr</code>'.

This change needs to be applied to AU Core Requester and Responder CapabilityStatements.

Reply:
Procedure is SHALL for multipleOr on status. No change.

Reply:
fixed.